doc/userguide/install.rst

```diff
@@ -164,211 +164,40 @@ and will present you with a ready-to-run (configured and set-up) Suricata.
 Binary packages
 ---------------
 
-Ubuntu from Personal Package Archives (PPA)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. toctree::
+   :maxdepth: 1
 
-For Ubuntu, OISF maintains a PPA ``suricata-stable`` that always contains the
-latest stable release.
+   install/ubuntu
+   install/debian
+   install/rpm
+   install/other
 
-.. note:: The following instructions require ``sudo`` to be installed.
-
-Setup to install the latest stable Suricata::
-
-    sudo apt-get install software-properties-common
-    sudo add-apt-repository ppa:oisf/suricata-stable
-    sudo apt-get update
-
-Then, you can install the latest stable with::
-
-    sudo apt-get install suricata
-
-After installing you can proceed to the :ref:`Basic setup`.
-
-`OISF launchpad: suricata-stable <https://launchpad.net/~oisf/+archive/suricata-stable>`_.
-
-Upgrading
-"""""""""
-
-To upgrade::
-
-    sudo apt-get update
-    sudo apt-get upgrade suricata
-
-Remove
-""""""
-
-To remove Suricata from your system::
-
-    sudo apt-get remove suricata
-
-
-
-Getting Debug or Pre-release Versions
-"""""""""""""""""""""""""""""""""""""
-
-.. note:: The following instructions require ``sudo`` to be installed.
+Suricata is available on various distributions as binary
+packages. These offer a convenient way to install and manage Suricata
+without compiling from source.
 
-If you want Suricata with built-in (enabled) debugging, you can install the
-debug package::
+For Ubuntu systems:
```

Comment:
Could we maybe use bold, or some formatting, to help differentiate each of those lines that indicate a system-family?

Reply:
Like...

**For Ubuntu systems**:

&nbsp;&nbsp;&nbsp;&nbsp;See Ubuntu Package Installation for detailed instructions on installing from PPA repositories.

**For Debian system**:

&nbsp;&nbsp;&nbsp;&nbsp;See Debian Package Installation for detailed instructions on installing from official repositories and backports.